porting/index.md

```diff
@@ -62,15 +69,54 @@ You can also specify `-j[num]` as it will fetch the files simultaneously. This d
 
 ### Put parts together
 
-Now you need to put all the parts for your device together, and since our tree is based on LineageOS for `halium-7.1` or CyanogenMod for `halium-5.1` you can use the device files they provide.
+Now you need to put all the parts for your device together, and since our tree is based on LineageOS for `halium-7.1` or CyanogenMod for `halium-5.1` you can use the device files they provide. Take care to switch the branch selection in Githubs webpage so you download the correct branches!
 
 Parts that are needed:
-- Device tree
-- Kernel source
+- Device tree (from [LineageOS](https://github.com/LineageOS))
+- Kernel source (also from LineageOS)
 - Vendor tree (e.g. from [TheMuppets](https://github.com/TheMuppets))
 
 You might have to check `lineage.dependencies`/`cm.dependencies` that is included in every LineageOS/CyanogenMod device repo for additional repositories.
 
+### Create a local manifest ###
+
+Create additional entries for the repo tool to download the required parts automatically:
+
+```
+cd <BUILDDIR>/.repo && mkdir local_manifests && cd local_manifests
+gedit <VENDOR>_<CODENAME>.xml. 
+```
+
+The repo tool will accept additional repositories to be synced on top of the ones defined by Halium already. Since you are porting for a new device, you also need to do this work. Later on your local manifest entries can become part of a central repository for device ports.
+
+For the time being, add entries to the created xml file as follows:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<!-- Example for a device with one specific vendor repo, and 2 repos for the device, a common and a specific one. At the end, a common kernel repo -->
+  <project path="vendor/samsung" name="proprietary_vendor_samsung" remote="them" />
+  <project name="android_device_samsung_smdk4412-common" path="device/samsung/smdk4412-common" remote="los" />
+  <project name="android_device_samsung_i9300" path="device/samsung/i9300" remote="los" />
+  <project name="android_kernel_samsung_smdk4412" path="kernel/samsung/smdk4412" remote="los" />
+  
+  <!-- Example for additional repos gathered from the dependencies files in the device or vendor repo -->
+  <project name="android_hardware_samsung" path="hardware/samsung" remote="los" />
+  <project name="android_external_stlport" path="external/stlport" remote="los" />
+
+</manifest>
+```
+
+The remote properties "los" and "them" are shortcuts for the already defined remotes in the Halium manifest.xml, which can be reviewed by looking into manifest.xml in the .repo directory. As default they point to 14.1 or 12.1 branches, respectively. You can create references to new remotes by adding the following snippet in front of any project definition:
```

Comment:
14.1 => 7.1
12.1 => 5.1

Then maybe link the manifest.xml directly and not just refer to it.
And the filename is also actually default.xml which gets symlinked to .repo/manifest.xml from .repo/manifests/default.xml